.circleci/config.yml

```diff
@@ -20,22 +25,48 @@ jobs:
           command: npm run test:ci
       - run:
           name: Install codecov
-          command: npm install -g codecov
+          command: sudo npm install -g codecov
```

Comment:
i presume image has nonroot user?

Reply:
I think so. This issue came up after switching to `circleci/node:10.16.1` docker image